### ISSUE TITLE
Fix username

### ISSUE THREAD

**Issue by dawipayandas** [lugnitdgp/SQLifyHUB#1]
### Fix username
Show proper username after login. It shows 'postgres' for every user.
![Screenshot from 2024-10-01 19-00-34](https://github.com/user-attachments/assets/9cc1ed4f-ad45-4c58-8d9a-09f72b32f616)

**Comment by jmiquis** [comment]
Hi there! Do you assign issues or straight PR?
thnx!

**Comment by dawipayandas** [comment]
> Hi there! Do you assign issues or straight PR? thnx!

In case you want to contribute, we'd assign the issue to you

**Comment by GitKimUser** [comment]
Hi there! Can I contribute this issue?

**Comment by dawipayandas** [comment]
> Hi there! Can I contribute this issue?

Yes, sure.

**Comment by Pudi-Sravan** [comment]
Hey, @GitKimUser how is it going,could you please share your progress,if you face any difficulty you can ask us anytime.

**Comment by GitKimUser** [comment]
Yes, hello, developer. I'm GitKimUser. Currently, I'm a college student, and I haven't been able to pay much attention to preparing for the exam. It's not much progress yet, and it's my first time contributing, so I'm not good at it. If you tell me when you need to promote until then, I'll try to do it before then. Currently, I can do PR as late as possible in a month from now. If you need to promote a month in advance from now, please let me know and I'll try to proceed. Thank you.
-----Original Message-----

**Comment by GitKimUser** [comment]
Hello developer. I'm sorry but I can't proceed quickly because there are other issues in progress. Thank you for assigned, but I don't think I can proceed. I'm sorry.
-----Original Message-----

**Comment by GitKimUser** [comment]
We will contribute to a good project soon. I'm so sorry.

**Comment by Pudi-Sravan** [comment]
No worries @GitKimUser you can take your time we will accept the pull requests till the month end

**Comment by AdityaJadhav9423** [comment]
Hey I want to work on this issue

**Comment by Pudi-Sravan** [comment]
hey @AdityaJadhav9423 we will assign you this soon ,you can start working on it.
